src/amo/components/ScreenShots/styles.scss

```diff
@@ -1,5 +1,7 @@
 @import '~amo/css/styles';
 
+/* stylelint-disable selector-class-pattern */
```

Comment:
does this disable the rule for the entire file?

Reply:
Yes, it does. We have three different selectors in this file that break the rule. They are needed for a specific third party component. I thought it simplest to just disable the rule for the whole file, rather than add three individual rule overrides. This rule is not really that important, but nice to have.